moveit_ros/planning/plan_execution/src/plan_execution.cpp

```diff
@@ -157,6 +157,31 @@ void plan_execution::PlanExecution::planAndExecute(ExecutableMotionPlan& plan,
   }
 }
 
+void plan_execution::PlanExecution::execute(ExecutableMotionPlan& plan)
```

Comment:
I don't yet understand, why we need a new method for this. `executeAndMonitor()` could be called directly, couldn't it?

Reply:
In theory I agree with you.

The reasoning behind this request is that `executeAndMonitor` is currently private and assumes that the monitor and planning scene of the argument have been setup properly.
It's actually a helper function, not a public interface.
The proposed method provides a public wrapper that imitates the API of `planAndExecute`.

Would you propose a different way to get the interface?